OpenKeychain/src/main/java/org/sufficientlysecure/keychain/remote/ui/RemoteSelectPubKeyActivity.java

```diff
@@ -103,7 +103,7 @@ protected void onCreate(Bundle savedInstanceState) {
                     public void onClick(View v) {
                         // add key ids to params Bundle for new request
                         Intent resultData = extras.getParcelable(EXTRA_DATA);
-                        resultData.putExtra(OpenPgpApi.EXTRA_KEY_IDS,
+                        resultData.putExtra(OpenPgpApi.EXTRA_KEY_IDS_SELECTED,
```

Comment:
Please correct me, but I thought that ``EXTRA_USER_IDS`` is removed for the second execution. Then only ``EXTRA_KEY_IDS`` is available and thus non-ambiguous.